.github/workflows/ci-build-binaries.yml

```diff
@@ -0,0 +1,32 @@
+name: Build binaries
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-binaries:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Fetch git tags
+      run: |
+        git fetch --prune --unshallow --tags
```

Comment:
why do we need this step when building binaries?

Reply:
also, when we do need tags, can we achieve the same effect with this setting?
```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```

Reply:
I think if we want to pull tags as part of `checkout` action, only option from my understanding is to set `fetch-depth: 0` which will also fetch all the commit history of all branches which will increase build time. I'm unable to find an option for just pulling the tags without commit history.

Reply:
maybe it's worth creating a PR for the official action to add a new parameter like `tags:`